File: src/main/java/br/com/svbgabriel/gerenciadortarefas/repositories/RepositoryUsuario.java
package br.com.svbgabriel.gerenciadortarefas.repositories;

import org.springframework.data.jpa.repository.JpaRepository;

import br.com.svbgabriel.gerenciadortarefas.models.Usuario;

public interface RepositoryUsuario extends JpaRepository<Usuario, Long> {
	Usuario findByEmail(String email);
}

File: src/main/kotlin/br/com/svbgabriel/gerenciadortarefas/repositories/RepositoryUsuario.kt
package br.com.svbgabriel.gerenciadortarefas.repositories

import org.springframework.data.jpa.repository.JpaRepository
import br.com.svbgabriel.gerenciadortarefas.models.Usuario

interface RepositoryUsuario : JpaRepository<Usuario, Long> {
	fun findByEmail(email: String): Usuario?
}